megamek/src/megamek/client/ui/swing/CommonMenuBar.java

```diff
@@ -826,11 +431,7 @@ private JMenuItem createMenuItem(JMenu m, String label, String command) {
      */
     public void actionPerformed(ActionEvent event) {
         // Pass the action on to each of our listeners.
-        Enumeration<ActionListener> iter = actionListeners.elements();
-        while (iter.hasMoreElements()) {
-            ActionListener listener = iter.nextElement();
-            listener.actionPerformed(event);
-        }
+        actionListeners.stream().forEach(l -> l.actionPerformed(event));
```

Comment:
Why not just use a forEach instead of streaming it?

Reply:
The joys of discovery :D